lib/smooth/core/network/Wifi.cpp

```diff
@@ -155,12 +157,17 @@ namespace smooth::core::network
             }
             else if (event_id == WIFI_EVENT_AP_START)
             {
-                wifi->ip.addr = 0xC0A80401; // 192.168.4.1
+                //wifi->ip.addr = 0xC0A80401; // 192.168.4.1
+                wifi->ip_info.ip.addr = 0xC0A80401; // 192.168.4.1
                 publish_status(true, true);
             }
             else if (event_id == WIFI_EVENT_AP_STOP)
             {
-                wifi->ip.addr = 0;
+                //wifi->ip.addr = 0;
```

Comment:
this too

Reply:
I believe I have removed all comments in question (and maybe one more)